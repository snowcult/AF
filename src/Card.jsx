const Card = function ({ id, email, name }) {
  return (
    <div className="tc  white bg-black-50  dib br3 pa2 ma3 grow bw2 shadow-4 content-center ">
      <img
        src={`https://robohash.org/${id}?size=220x220&set=set2`}
        alt="robot"
      />
      <div>
        <h2>{name}</h2>
        <p>{email}</p>
      </div>
    </div>
  );
};
export default Card;
