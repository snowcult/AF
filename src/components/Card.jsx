import PropTypes from 'prop-types';
const Card = function ({ id, email, name }) {
  return (
    <div className="c  white bg-black-50  dib br3 pa2 ma3 grow bw2 shadow-4">
      <img
        src={`https://robohash.org/${id}?size=220x220&set=set2`}
        alt="robot"
      />
      <div>
        <h2>{name}</h2>
        <p>{email}</p>
      </div>
    </div>
  );
};
Card.propTypes = {
  email: PropTypes.string,
  name: PropTypes.string,
  id: PropTypes.number,
};
export default Card;
