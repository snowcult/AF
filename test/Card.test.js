import { shallow } from 'enzyme';
import React from 'react';
import Card from '../src/components/Card.jsx';
import toJson from 'enzyme-to-json';

it('expect to render Card comp', () => {
  const wrapper = shallow(<Card />);
  expect(wrapper).toMatchSnapshot();
});
