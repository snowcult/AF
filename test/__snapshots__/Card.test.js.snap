// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to render Card comp 1`] = `
<div
  className="tc  white bg-black-50  dib br3 pa2 ma3 grow bw2 shadow-3"
>
  <img
    alt="robot"
    src="https://robohash.org/undefined?size=220x220&set=set2"
    style={
      {
        "height": "220px",
        "objectFit": "cover",
        "width": "220px",
      }
    }
  />
  <div>
    <h2 />
    <p />
  </div>
</div>
`;
